cypress/e2e/dfi.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { exchanges } from "../fixture/dfi.config";
+
+const viewports = ["iphone-xr", "ipad-2", "macbook-16"];
+
+viewports.forEach((viewport) => {
+  context(`/explore/dfi on ${viewport}`, () => {
+    beforeEach(() => {
+      cy.visit("/explore/dfi");
+      cy.viewport(<Cypress.ViewportPreset>viewport);
+    });
+
+    it("should have Explore DFI section visible and expected text", () => {
+      it("should be visible with expected text", () => {
+        cy.checkElementVisibilityAndText(
+          "section-title-explore-dfi",
+          "EXPLORE THE POWER OF DFI"
+        );
+        cy.checkElementVisibilityAndText(
+          "section-sub-title",
+          "Experience DeFi with DFI"
+        );
+        cy.checkElementVisibilityAndText(
+          "section-desc",
+          "Our native token unlocks the power of decentralized finance, giving you access to a growing ecosystem of cutting-edge DeFi tools and innovative blockchain applications."
+        );
+      });
+    });
+
+    it("should have Start exploring section clickable", () => {
+      if (viewport === "macbook-16") {
+        cy.findByTestId("start-exploring-button").click({
+          scrollBehavior: false,
+        });
+        cy.url().should("include", "#statistics-display-dfi");
```

Comment:
assert if `statistics-display-dfi` section is visible after start exploring button is clicked